specs/Freeze.md

```diff
@@ -0,0 +1,155 @@
+# Background
+Edge browser has a sleeping tab feature which reduces resource usage when the tab is in the background. We are introducing WebView2 APIs
+to access this feature so that invisible WebView can use less resources. We'd appreciate your feedback.
+
+
+# Description
+You may call the `TryFeezeAsync` API to have the WebView2 consume less memory. This is useful when your Win32 app becomes invisible, or when your Universal Windows Platform app is suspended, during the suspended event handler before completing the suspended event.
```

Comment:
My comments are in here https://github.com/MicrosoftEdge/WebView2Feedback/pull/621/files#r533302500

To me "freeze" implies "prevent any changes to the state of an object", also "freeze" has somewhat a negative connotation, e.g. "my computer froze" or "app freezes".
The sample below look confusing:
```cs
async protected void OnSuspending(object sender, SuspendingEventArgs args)
{
    SuspendingDeferral ... = ...Suspending...

    await ....TryFreezeAsync(); 
}
```

Since the WebView2 control is punted as **the web browser** control for .NET, I think it may worthwhile chatting with @terrajobst about a formal API review to make sure the API aligns with the .NET vocab.

Reply:
I'm going to separate your comments into 3 issues here to discuss in the API review: the name, whether a Try method also requires a non-Try method, and whether a method pair must match in terms of Try or Async. And I'll send an email to Immo